packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.js

```diff
@@ -2690,6 +2692,234 @@ describe('ReactHooksWithNoopRenderer', () => {
     });
   });
 
+  describe('useMutationEffect', () => {
+    // @gate experimental || www
```

Comment:
What does `@gate` annotation and `www` meaning?  
Also I've curious about [@gate FIXME](https://github.com/facebook/react/blob/cae635054e17a6f107a39d328649137b83f25972/packages/react-reconciler/src/__tests__/ReactNewContext-test.js#L1085). 🤔

Reply:
`@gate` is a custom annotation we have to gate tests to only pass in certain conditions. If the condition fails, we verify that the test fails. `www` is a special build variant that we use for internal testing.